src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -59,40 +59,43 @@ public void start(Map<String, String> props, JestClient client) {
       log.info("Starting ElasticsearchSinkTask.");
 
       ElasticsearchSinkConnectorConfig config = new ElasticsearchSinkConnectorConfig(props);
-      String type = config.getString(ElasticsearchSinkConnectorConfig.TYPE_NAME_CONFIG);
-      boolean ignoreKey =
+      final String type = config.getString(ElasticsearchSinkConnectorConfig.TYPE_NAME_CONFIG);
```

Comment:
That's a considerable shift to using `final` keyword for local variables. And although this might be a lost battle, I think a discussion would be nice to take place regarding this pattern. Note: I strongly disagree with this pattern.

Any particular reason this is applied here, in this particular PR? Since it's such a massive change, probably belongs in a separate refactoring PR, if it has to be applied, anyways.